roboflow/util/image_utils.py

```diff
@@ -1,10 +1,40 @@
+import os
+import urllib
+
 import requests
 
 
-# Checks if hosted image hurl is a valid image
+def check_image_path(image_path):
+    """
+    Check whether a local OR hosted image path is valid
+    :param image_path: local path or URL of image
+    :returns: Boolean
+    """
+    return os.path.exists(image_path) or check_image_url(image_path)
+
+
 def check_image_url(url):
-    if "http://" in url or "https://" in url:
-        r = requests.head(url)
-        return r.status_code == requests.codes.ok
+    """
+    Check whether a hosted image path is valid
+    :param url: URL of image
+    :returns: Boolean
+    """
+    if urllib.parse.urlparse(url).scheme not in (
+        "http",
+        "https",
+    ):
+        return False
+
+    r = requests.head(url)
+    return r.status_code == requests.codes.ok
+
 
-    return False
+def validate_image_path(image_path):
+    """
+    Validate whether a local OR hosted image path is valid
+    :param image_path: local path or URL of image
+    :returns: None
+    :raises Exception: Image path is not valid
+    """
+    if not check_image_path(image_path):
+        raise Exception(f"Image does not exist at {image_path}!")
```

Comment:
I noticed a lot of code duplication in the codebase for validating an `image_path`, so I made a couple generic helpers for this. We could refactor much of the codebase to use these helpers in the future.